independent-publisher-connectors/apiDefinition.swagger.json

```diff
@@ -0,0 +1,873 @@
+          }
+        }
+      }
+    },
+    "/update-template": {
+      "post": {
+        "description": "This action updates the specified PDF template.",
+        "summary": "Update template",
+        "tags": [
+          "Template Management API"
+        ],
+        "operationId": "Update-template",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "description": "The body.",
+            "schema": {
+              "$ref": "#/definitions/UpdateTemplateRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns status and output file.",
+            "schema": {
+              "$ref": "#/definitions/ResponseUpdateTemplate"
+            },
+            "headers": {}
+          }
+        }
+      }
+    },
+    "/delete-template": {
+      "get": {
+        "description": "This action deletes a template.",
+        "summary": "Delete template",
+        "tags": [
+          "Template Management API"
+        ],
+        "operationId": "Delete-template",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "template_id",
+            "in": "query",
+            "x-ms-summary": "Template ID",
+            "required": true,
+            "type": "string",
+            "description": "The PDF template ID."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns status and output file.",
+            "schema": {
+              "$ref": "#/definitions/ResponseSuccessDeleteTemplate"
+            },
+            "headers": {}
+          }
+        }
+      }
+    },
+    "/create-editor-session": {
+      "post": {
+        "description": "This action creates a new PDF editor session. The PDF editor url can be embedded into an IFrame.",
+        "summary": "Create editor session",
+        "tags": [
+          "Template Management API"
+        ],
+        "operationId": "Create-editor-session",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "description": "The body.",
+            "schema": {
+              "$ref": "#/definitions/CreateEditorSessionRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns status and editor URL.",
+            "schema": {
+              "$ref": "#/definitions/ResponseSuccessCreateNewEditorSession"
+            },
+            "headers": {}

```

Comment:
@troystaylor Empty property values **in the swagger** can be removed until unless you have a specific reason to keep them.